### ISSUE TITLE
RealmSchema not resolved in  'io.realm:realm-android:0.82.1'

### ISSUE THREAD

**Issue by Jagat-sdrc** [realm/realm-java#4285]
I am compiling with io.realm:realm-android:0.82.1
and when i am using the Migration the RealmSchema is not working ,
So any can resolve my problem
```

#### Version of Realm and tooling
Realm version(s): ?  io.realm:realm-android:0.82.1

Realm sync feature enabled: yes/no

Android Studio version: ? 2.3

Which Android version and device: ? 25 , Samsung

**Comment by Zhuinden** [comment]
Yeah, if this is a new project, and your MinSDK is 19, then please use Realm 3.0.0, otherwise Realm 1.2.0

**Comment by kneth** [comment]
My best advice is that you upgrade. We have added many new features and fixed bugs in the last 1.5 years ;-)
